Fix the p2p references from default unit test project

vijayrkn opened the pull request:
@ahmelsayed  

/cc @lindydonna 

Fix for https://github.com/Azure/azure-functions-vs-build-sdk/issues/94

dnfclas commented:
@vijayrkn,
Thanks for having already signed the Contribution License Agreement. Your agreement was validated by .NET Foundation. We will now review your pull request.
_Thanks,
.NET Foundation Pull Request Bot_

vijayrkn commented:
Updated the samples to have the following additional scenarios:
1. Function app project with references to Classic class lib & Sdk class lib
2. Classic unit test project with reference to Function app and running the tests from this project.
3. SDK style unit test projects (mstest and xunit) with reference to function app and running tests from these test projects.

Also updated the package version to 1.0.2
